spec/models/api_v1_post_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe ApiV1Post, type: :model do
```

Comment:
get '/api/v1/posts'
をしていますが、今のままだとデータが何も帰ってこないのでテストとしては不十分ですね！

before { create(:post) }を追加してテスト用のデータ事前に一つ作っておくと良さそうです！

Reply:
こちらよくみるとmodel　specではなくrequest specだと思うので、ディレクトリーも変更した方が良さそうですね！spec/requests/api_v1_post_spec.rb
```suggestion
RSpec.describe ApiV1Post, type: :requrest do
```